dev_guide/deployments.adoc

```diff
@@ -247,7 +247,7 @@ triggers:
 === Image Change Trigger
 
 The ImageChange trigger results in a new deployment whenever the value of an
-image stream tag changes.
+image stream tag changes either by a build or because it was imported.
```

Comment:
Correct. CCT is responsible for triggering new deployments on pod template
changes. If it's on, **any template change** should trigger a deployment,
if it's off, template changes will not trigger deployments automatically.
Essentially, CCT is (almost) a global on/off for a DC. ICTs set to
automatc: true will always update images in templates, automatic: false
will update only for the initial deployment so people wont get stuck with
ImagePullBackOff errors

On Fri, May 27, 2016 at 11:35 PM, Ben Parees <notifications@github.com>
wrote:

> In dev_guide/deployments.adoc
> <https://github.com/openshift/openshift-docs/pull/2150#discussion_r64969500>
> :
>
> > @@ -247,7 +247,7 @@ triggers:
> >  === Image Change Trigger
> >
> >  The ImageChange trigger results in a new deployment whenever the value of an
> > -image stream tag changes.
> > +image stream tag changes either by a build or because it was imported.
>
> yeah, without the CCT (but with an ICT in automatic mode) the DC will be
> updated w/ the new image information (at a minimum this will happen when a
> deployment is explicilty requested), but no deployment will be triggered
> automatically. I believe.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/openshift-docs/pull/2150/files/97c25356b03bef326537fbf7aa454100ec58a53a#r64969500>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ADuFf7WSZ9mITHQhBAKpYFiw3cohCHuVks5qF2OlgaJpZM4ImsqM>
> .
>